code/parse/sexp.cpp

```diff
@@ -20448,6 +20448,7 @@ void sexp_debug(int node)
 	node = CDR(node); 
 	Assertion (node >= 0, "No message defined in debug SEXP"); 
 	id = CTEXT(node);
+	strcpy_s(temp_buf, id);
```

Comment:
Should `sexp_replace_variable_names_with_values` be called on this buffer if it's not a valid message id? I think that would make sense since the message variant also does this.